packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -467,21 +467,22 @@ contract OPContractsManager is ISemver {
         virtual
         returns (bytes memory)
     {
-        bytes4 selector = ISystemConfig.initialize.selector;
         (IResourceMetering.ResourceConfig memory referenceResourceConfig, ISystemConfig.Addresses memory opChainAddrs) =
-            defaultSystemConfigParams(selector, _input, _output);
-
-        return abi.encodeWithSelector(
-            selector,
-            _input.roles.systemConfigOwner,
-            _input.basefeeScalar,
-            _input.blobBasefeeScalar,
-            bytes32(uint256(uint160(_input.roles.batcher))), // batcherHash
-            _input.gasLimit,
-            _input.roles.unsafeBlockSigner,
-            referenceResourceConfig,
-            chainIdToBatchInboxAddress(_input.l2ChainId),
-            opChainAddrs
+            defaultSystemConfigParams(_input, _output);
+
+        return abi.encodeCall(
+            ISystemConfig.initialize,
```

Comment:
This also implies we should be able to clean up all the other `encode*` methods that take `bytes4 _selector` right? Happy to track that in a separate issue/PR if you prefer

Reply:
oh true... I didn't realize that they act the same way, had assumed it was due to the old aim of being backwards compatible... 

I say we just get them out now, I'll make another commit here.

Reply:
c00b72421176

Reply:
Sweet, resolving this thread